src/psyclone/tests/psyir/nodes/call_test.py

```diff
@@ -43,11 +43,12 @@
 from psyclone.parse import ModuleManager
 from psyclone.psyir.nodes import (
     ArrayReference, Assignment, BinaryOperation, Call, CodeBlock, Literal,
-    Reference, Routine, Schedule)
+    Reference, Routine, Schedule, Node)
 from psyclone.psyir.nodes.node import colored
 from psyclone.psyir.symbols import (
     ArrayType, INTEGER_TYPE, DataSymbol, NoType, RoutineSymbol, REAL_TYPE,
     SymbolError, UnsupportedFortranType)
```

Comment:
We aspire to have all imports in alphabetical order, at least by module name. Therefore, please could you move this one up to L47.

Reply:
OK. I assume you're talking about "Node". The new push will also be reformatted to have only one line per object.

Reply:
Understood: The entire import/form should be alphabetically ordered.
Done.

Reply:
Thanks :-)